metricbeat/module/redis/redis.go

```diff
@@ -0,0 +1,77 @@
+/*
+Shared Redis functionality for the metric sets
```

Comment:
Package comments should start with `Package <name> ....`. Otherwise you get a golint warning. 
https://github.com/golang/go/wiki/CodeReviewComments#package-comments